whipper/program/utils.py

```diff
@@ -34,8 +30,9 @@ def load_device(device):
 
 def unmount_device(device):
     """
-    Unmount the given device if it is mounted, as happens with automounted
-    data tracks.
+    Unmount the given device if it is mounted.
+
+    This usueally happens with automounted data tracks.
```

Comment:
A small typo i guess 😉
```suggestion
    This usually happens with automounted data tracks.
```